enclave_manager/Dockerfile

```diff
@@ -92,7 +87,13 @@ RUN OPENSSL_VER=1.1.1g \
  && make test \
  && make install -j$THREADS
 
-
+RUN if [ "$DISTRO" = "bionic" ] ; then \
+      mkdir /usr/local/lib64; \
```

Comment:
This looks weird that Dockerfile creates these directories.  This should be created during installation of packages.

Reply:
This is kind of workaround as to have a single Dockerfile for bionic and centos has its own limitations!

Reply:
what error where you getting if you don't have this. We didn't require this step earlier with ubuntu.

Reply:
Yes, this step was not needed for ubuntu. In centos, it has packages installed in /usr/local/lib64. which have to copied in build_image. But /usr/local/lib64 doesn't exist in ubuntu, so creating a empty directory in ubuntu, so that COPY command in build_image doesn't fail.